forc/src/cli/commands/test.rs

```diff
@@ -32,6 +32,9 @@ pub struct Command {
     /// When specified, only tests containing the given string will be executed.
     pub filter: Option<String>,
     #[clap(long)]
+    /// When specified, only the test exactly matching the given string will be executed.
+    pub exact: bool,
```

Comment:
Maybe this should be `filter_exact` to make its association with the filter a little more clear?

Reply:
Good idea!